QA/README.md

```diff
@@ -0,0 +1,79 @@
+# CherryTwist - Quality Assurance
+
+This document provides overview of "Cherrytwist" quality assurance and testing activities. Some of them are in progress and some are to be started.
```

Comment:
This document provides an overview of the quality assurance and testing activities that are carried out on the Cherrytwist platform
- basically do not like Cherrytwist in quotes.